OrbitGl/App.cpp

```diff
@@ -1034,11 +1035,46 @@ DataView* OrbitApp::GetOrCreateDataView(DataViewType type) {
     case DataViewType::kAll:
       FATAL("DataViewType::kAll should not be used with the factory.");
 
+    case DataViewType::kTracepoints:
+      if (!tracepoints_data_view_) {
+        tracepoints_data_view_ = std::make_unique<TracepointsDataView>();
+
+        if (!options_.grpc_server_address.empty()) {
+          grpc::ChannelArguments channel_arguments;
+
+          channel_arguments.SetMaxReceiveMessageSize(std::numeric_limits<int32_t>::max());
+          grpc_channel_ = grpc::CreateCustomChannel(
+              options_.grpc_server_address, grpc::InsecureChannelCredentials(), channel_arguments);
+          if (!grpc_channel_) {
+            ERROR("Unable to create GRPC channel to %s", options_.grpc_server_address);
+          } else {
+            tracepoint_manager_ = TracepointManager::Create(grpc_channel_);
```

Comment:
also probably create this where all other service clients are created - in PostInit